Documentation/sourcebuild-user-onboarding-guide/bootstrapping.md

```diff
@@ -0,0 +1,14 @@
+# Bootstrapping the source-build
+
+## Bootstrapping Subsequent .NET release
+Subsequent versions of .NET Core, when being built for an already bootstrapped distro do not need to follow the bootstrap process. Subsequent versions will use the N-1 version of the toolset from the archive to build. Note, N-1 in this context means the previous version of .NET Core that was included in the distro archive.
+As an example, to build SDK “N” (6.0.101) using “N-1” (6.0.100). 
+
+For the major release (7.0 in the example), the bootstrap process needs to be re-executed.
+
+## Bootstrapping New RID (Arch/OS)
+Refer to this guide to learn about the RID - https://learn.microsoft.com/en-us/dotnet/core/rid-catalog#linux-rids.  
+Source-build is required to build on a single machine with no internet access. This means that we build targeting a single RID, usually the non-portable RID for the build machines (like rhel.7-x64). We do support building portable (linux-x64) as well - this is useful for bootstrapping new distributions.  
+
+## Bootstrapping Banana Distro’s 
```

Comment:
```suggestion
## Bootstrapping new distros 
```

Our banana terminology is a bit inside baseball.

Reply:
Do you suggest any other terminology as New Distro's can be confusing and does not essentially mean the Distro's which are not supported by .NET yet.

Reply:
Putting New Distro's for now. I will update after getting suggestions.